packages/cli/src/commands/publish/publishPackages.ts

```diff
@@ -71,33 +90,30 @@ export default async function publishPackages({
     warn("No unpublished packages to publish");
   }
 
-  let promises: Promise<{
-    name: string;
-    newVersion: string;
-    published: boolean;
-  }>[] = [];
-
-  for (let pkgInfo of unpublishedPackagesInfo) {
-    let pkg = packagesByName.get(pkgInfo.name)!;
-    promises.push(
-      publishAPackage(
-        pkg,
-        access,
-        twoFactorState,
-        getReleaseTag(pkgInfo, preState, tag)
-      )
-    );
-  }
-
-  return Promise.all(promises);
+  return Promise.all(
+    unpublishedPackagesInfo.map(pkgInfo =>
+      // If there are many packages to be published, it's better to limit the
+      // concurrency to avoid unwanted errors, for example from NPM.
```

Comment:
```suggestion
      // concurrency to avoid unwanted errors, for example from npm.
```

Nit: npm is [lower case](https://css-tricks.com/start-sentence-npm/) 😅